flutter_mobx/CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
+## 2.2.0+1
+ - renamed `Observer.withChild` to `Observer.excludedChild`
+`
 ## 2.2.0
- - `Observer` is updated with the new `Observer.withChild` constructor, so you can exclude child branch from the re-rendering. - [@subzero911](https://github.com/subzero911) \
-   In case if you use `Builder.withChild`, you should provide two parameters: `builderWithChild` and `child`:
+ - `Observer` is updated with the new `Observer.excludedChild` constructor, so you can exclude child branch from the re-rendering. - [@subzero911](https://github.com/subzero911) \
+   In case if you use `Observer.excludedChild`, you should provide two parameters: `builderWithChild` and `child`:
    ```dart
-   Observer.withChild(
+   Observer.excludedChild(
```

Comment:
how about withBuiltChild to signify the state of the child. Excluded child is still not revealing the true intent. ?

Reply:
Good point! Will rename